### PR TITLE
Address a couple of deprecations

### PR DESCRIPTION
**Pull request by jamesstout**
What does this implement/fix? Explain your changes.
---------------------------------------------------
Address a couple of deprecations:

1. update for AbsoluteToNanoseconds deprecation
2. update for descriptionWithCalendarFormat deprecation

Also a couple of main thread fixes.

Does this close any currently open issues?
------------------------------------------
No

Any other comments?
-------------------
I added tests for the new methods.

Where has this been tested?
---------------------------
**Devices/Simulators:** iMac19,1

**macOS Version:** 10.15.5

**Sequel-Ace Version:** latest dev

### CODE REVIEW

File: Source/SPDataImport.m [hunk 1]
```diff
@@ -371,6 +373,16 @@ - (void)startSQLImportProcessWithFile:(NSString *)filename
  */
 - (void)importSQLFile:(NSString *)filename
 {
+	SPLog(@"Staring import....");
```
Comment (gboudreau) [inline Source/SPDataImport.m:376]: Starting

File: Source/SPDataImport.m [hunk 2]
```diff
@@ -371,6 +373,16 @@ - (void)startSQLImportProcessWithFile:(NSString *)filename
  */
 - (void)importSQLFile:(NSString *)filename
 {
+	SPLog(@"Starting import....");
+	
+//	TODO: this is slowwwwwwww
```
Comment (Kaspik) [inline Source/SPDataImport.m:378]: This should be removed before merging?

Comment (jamesstout) [reply Source/SPDataImport.m:378]: Maybe, it's still TODO though. Probably the process should be to raise an issue rather than leave a TODO in there.

Comment (Kaspik) [reply Source/SPDataImport.m:378]: Yeah, I think "TODO"s shouldn't be merged to master, either put a warning there or create an issue :)

**Pull request merged**